lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1015,6 +1017,65 @@ def config_apn_correction(self, cmd):
         else:
             print("APN Correction failed")
 
+    def config_pipelined_managed_tbl0(self, cmd):
+        """
+        Configure the flag to control ovs table=0 using pipelined
+
+        Args:
+          cmd: Specify how to configure pipelined_managed_tbl0 on AGW,
+          should be one of
+            enable: Enable pipelined_managed_tbl0 feature, do nothing if already enabled
+            disable: Disable pipelined_managed_tbl0 feature, do nothing if already disabled
+
+        """
+        pipelined_managed_tbl0_cmd = ""
+        if cmd.name == MagmadUtil.pipelined_managed_tbl0_cmds.ENABLE.name:
+            pipelined_managed_tbl0_cmd = "sed -i \'s/pipelined_managed_tbl0: false/pipelined_managed_tbl0: true/g\' /etc/magma/spgw.yml"
+            ret_code = self.exec_command(
+                "sudo " + pipelined_managed_tbl0_cmd,
+            )
+
+            if ret_code == 0:
+                print("pipelined_managed_tbl0 enabled")
+
+        else:
+            pipelined_managed_tbl0_cmd = "sed -i \'s/pipelined_managed_tbl0: true/pipelined_managed_tbl0: false/g\' /etc/magma/spgw.yml"
+            ret_code = self.exec_command(
+                "sudo " + pipelined_managed_tbl0_cmd,
+            )
+
+            if ret_code == 0:
+                print("pipelined_managed_tbl0 disabled")
+
+    def config_enable5g_features(self, cmd):
+        """
+        Configure the flag to support 5G functionality to pipelined
+
+        Args:
+          cmd: Specify how to configure 5G functionality to pipelined on AGW,
+          should be one of
+            enable: Enable enable5g_features flag, do nothing if already enabled
+            disable: Disable enable5g_features flag, do nothing if already disabled
+
+        """
+        enable5g_features_cmd = ""
+        if cmd.name == MagmadUtil.enable5g_features_cmds.ENABLE.name:
+            enable5g_features_cmd = "sed -i \'s/enable5g_features: false/enable5g_features: true/g\' /etc/magma/pipelined.yml"
+            ret_code = self.exec_command(
+                "sudo " + enable5g_features_cmd,
+            )
+
+            if ret_code == 0:
+                print("enable5g_features enabled")
+        else:
+            enable5g_features_cmd = "sed -i \'s/enable5g_features: true/enable5g_features: false/g\' /etc/magma/pipelined.yml"
```

Comment:
Is there any downside of enabling enable5g_features and pipelined_managed_tbl0  by default?

Reply:
ok, lets enable enable5g_features for pipelined.
We have to keep pipelined_managed_tbl0 off by default for perf reason.

Reply:
@pshelar , Sorry for late response.. As Ulas suggested we have to make for generalizing the behavior for enable5g_features flags in yaml files and By default orc8r should drive the behavior. So that we had Commented the enable5g_features flag from all yaml files recently. (#11797 )

Reply:
You you enable it for magma dev-vm pipelineD config only? keep prod config as it is.
enable it here: https://github.com/magma/magma/blob/master/lte/gateway/configs/pipelined.yml

cc: @ulaskozat

Reply:
@pshelar We are configuring `enable5g_features` flag in side dev-vm under `/etc/magma/pipelined.yml` file which is always sync with `magma/lte/gateway/configs/pipelined.yml`. All S1AP test cases are following same procedure like editing yml files under dev-vm's `/etc/magma/` path.  Though we have `/etc/magma/pipelined.yml_prod` there is no such  `enable5g_features` flag under `/etc/magma/pipelined.yml_prod`

Reply:
@pshelar for changing `enable5g_features` default value to `true` in `magma/lte/gateway/configs/pipelined.yml` we need to take grant from Ulas before making changes.
CC: @ulaskozat Could you please help here to move further on this task.

Reply:
it is fine. @pshelar please add documentation for pipelined.yml vs. pipelined.yml_prod.

Reply:
sure.

Reply:
Sorry for the delay there was some confusion, now edited `enable5g_features` default value to `true` in `magma/lte/gateway/configs/pipelined.yml` as suggested by @pshelar and @ulaskozat .
@pshelar could you please have one more round of review and move this PR to further

Reply:
@GANESH-WAVELABS some of mandatory checks are failing, can you fix it.

Reply:
Thanks @pshelar , I fixed the mandatory checks which was failing earlier. PR is `ready2merge` now.